assets/src/ba_data/python/bastd/actor/spaz.py

```diff
@@ -1225,7 +1225,7 @@ def drop_bomb(self) -> stdbomb.Bomb | None:
         pos = self.node.position_forward
         vel = self.node.velocity
 
-        if self.land_mine_count > 0:
+        if self.land_mine_count >= 0:
```

Comment:
This looks like it will completely break the logic. Just wanting to echo @Dliwk in asking what the motivation is behind these > to >= changes?...